.github/cspell.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": "0.2",
+  "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+  "dictionaries": [
+    "vgv_allowed",
+    "vgv_forbidden"
+  ],
+  "dictionaryDefinitions": [
+    {
+      "name": "vgv_allowed",
+      "path": "https://raw.githubusercontent.com/verygoodopensource/very_good_dictionaries/main/allowed.txt",
+      "description": "Allowed VGV Spellings"
+    },
+    {
+      "name": "vgv_forbidden",
+      "path": "https://raw.githubusercontent.com/verygoodopensource/very_good_dictionaries/main/forbidden.txt",
+      "description": "Forbidden VGV Spellings"
+    }
+  ],
+  "useGitignore": true,
+  "words": [
+    "xcworkspace",
+    "xcassets",
+    "xcconfig",
+    "xcplayground",
+    "xcshareddata",
+    "xcscheme",
+    "xctest",
+    "xctool",
+    "xctrunner",
+    "xctemplate",
+    "xcodeproj",
+    "xcodeb",
```

Comment:
Should these be in the core allowed list?

/cc @scarletteliza

Reply:
go for it

Reply:
Can we remove them from here then?